std_msgs/Header header
vision_msgs/Detection2DArray detections
sensor_msgs/Image[] masks
